### PR TITLE
add scheduled signal

### PR DESCRIPTION
**Pull request by amshan15**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12008)

### CODE REVIEW

File: packages/experiments/package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "@uifabric/theme-samples": "^7.0.22",
     "@uifabric/utilities": "^7.12.1",
     "@uifabric/variants": "^7.1.12",
+    "core-js": "3",
```
Comment (ecraig12345) [inline packages/experiments/package.json]: We shouldn't add a dependency on core-js in this package. If you explain why you needed it, I can probably help find a workable alternative.

Comment (amshan15) [reply packages/experiments/package.json]: whoops that should not be there. I'll remove it and update the PR

Comment (ecraig12345) [reply packages/experiments/package.json]: Thanks! You'll need to update yarn.lock too.

Comment (amshan15) [reply packages/experiments/package.json]: Just updated. Thanks!

**Pull request merged**